core/cog-launcher.c

```diff
@@ -117,12 +117,39 @@ on_permission_request (G_GNUC_UNUSED WebKitWebView *web_view,
     return TRUE;
 }
 
+static gboolean
+on_load_failed_with_tls_errors (WebKitWebView       *web_view,
+                                char                *failing_uri,
+                                GTlsCertificate     *certificate,
+                                GTlsCertificateFlags errors,
+                                void                *user_data)
+{
+    CogShell *shell = COG_SHELL (user_data);
+    WebKitWebContext *context = webkit_web_view_get_context (web_view);
+
+    if (cog_shell_get_ignore_tls_errors (shell)) {
+        g_autoptr(SoupURI) uri = soup_uri_new (failing_uri);
+        g_message ("<%s> Ignoring TLS errors", failing_uri);
+        webkit_web_context_allow_tls_certificate_for_host (context,
```

Comment:
This is thought to add exceptions when the global policy is to fail in case of TLS errors. This patch is adding a global policy, so I trhink it should be easier and more efficient to use webkit_web_context_set_tls_errors_policy()

Reply:
I have updated the PR to set this web context option instead, thanks for the suggestion!